src/Chapter1.hs

```diff
@@ -438,7 +438,7 @@ Implement the function that takes an integer value and returns the next 'Int'.
 -}
 next :: Int -> Int
 next x = error "next: not implemented!"
-
+next_x = x + 1
```

Comment:
Unfortunately, this implementation still has `error`. When writing `next_x`, you don't introduce a variable called `x`. You need to add space between function name and argument. Also, you need to replace the `error "..."` with your implementation.